core/src/main/kotlin/discord4j/core/shard/ShardingClientBuilderExtensions.kt

```diff
@@ -0,0 +1,8 @@
+package discord4j.core.shard
+
+import discord4j.core.DiscordClientBuilder
+import discord4j.core.shard.ShardingClientBuilder
+import kotlinx.coroutines.reactive.awaitSingle
+
+
+suspend fun ShardingClientBuilder.create(): List<DiscordClientBuilder> = build().collectList().awaitSingle()
```

Comment:
This is why I said most of the internal extensions earlier seem pointless. It's not even used everywhere, and really I don't think it needs to. This seems just fine to have laying around.